### ISSUE TITLE
Q.: relation with com.jaspersoft.studio.babel.editor?

### ISSUE THREAD

**Issue by meden** [essiembre/eclipse-rbe#21]
Hello, I had `eclipse-rbe` and then I installed JasperStudio Plugin. It ended up with a duplicated resource bundle editor plugin. So, my question is: which is the relation between `com.essiembre.eclipse.i18n.resourcebundle` and `com.jaspersoft.studio.babel.editor`? They seem almost identical.

Thanks for your work, it is very useful.

**Comment by essiembre** [comment]
You are asking for a bit of history. :-) 

I started a rewrite of ecliplse-rbe a few years ago to make it more efficient and add a few things from what I learned building this plugin so far.  That code in progress ended up to be a donation to the official Eclipse Babel project, with the idea that it would become an official Eclipse plugin.

Given my lack of time, nobody else really took over that plugin under the Eclipse Babel umbrella while the currently published one (on SourceForge at the time) kept increasing in popularity.  So to satisfy loyal users of the original plugin, I revived it by transferring it to Github and made it work with newer Eclipse versions (plus added a few fixes).  Now this project is back on track with two new contributors working on fixes and new features (and I am thankful for that!). 

If not mistaken, Jasper took the open-source code donated to the Ecliplse Babel project and adapted it to their product.  They could have used this project's code instead, but going with what seemed destined to be an official plugin at some point made more sense at the time. 

Making a long story short, the code for both originally comes from the same author (i.e. myself) and that's why it looks/is the same.  The difference is they each come from a now different code base/source, and they probably made a few tweaks/additions of their own to better fit their product.

Does that answer?
